.github/workflows/docs.yml

```diff
@@ -0,0 +1,17 @@
+name: Docs build
+
+on: [push, pull_request]
+
+jobs:
+  docs build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      # TODO: Dependencies come from environment.yml.
+      # How should they be shared here? Install everything?
```

Comment:
Not sure of the best way to do this. I notice that some dependencies in `environment.yml` are repeated in `setup.py` as well, but know you are already thinking about breaking up dependencies with `extras_require`.

I think ideally `environment.yml` is used to setup a virtual environment with correct python version – maybe with some additional dependencies for running notebooks / building vitessce package (e.g., anndata, loompy, build). Development dependencies could then also be an `extras_requires` entry, importing contents from a standalone `requirements-dev.txt`, which can be pip installed here in isolation.

```bash
pip install -r requirements-dev.txt # just install dev dependencies (what to use in this workflow)
pip install -e .[dev] # install vitessce, dependencies, and dev dependencies
```

Maybe difficult to explain here. Happy to open a PR if interested.

Reply:
@manzt -- Yeah, if you can open up a follow-up PR, that'd be useful. Thanks.